Dockerfile

```diff
@@ -1,39 +1,33 @@
 FROM python:3.5
 RUN apt-get -y update && apt-get -y upgrade
-
 RUN apt-get install -y libpq-dev supervisor curl vim
-## BUILD JS STUFF
-RUN wget https://nodejs.org/dist/v10.1.0/node-v10.1.0-linux-x64.tar.xz -P /tmp && \
-  cd /tmp && tar xfvJ node-v10.1.0-linux-x64.tar.xz && \
-  ln -sf /tmp/node-v10.1.0-linux-x64/bin/node /usr/bin/node
 RUN useradd -m openslides
+
+## BUILD JS STUFF
+RUN wget https://nodejs.org/dist/v10.5.0/node-v10.5.0-linux-x64.tar.xz -P /tmp && \
+  cd /tmp && tar xfvJ node-v10.5.0-linux-x64.tar.xz && \
+  ln -sf /tmp/node-v10.5.0-linux-x64/bin/node /usr/bin/node
 RUN mkdir /app
-COPY package.json yarn.lock bower.json gulpfile.js /app/
 WORKDIR /app
+COPY . /app
 RUN chown -R openslides /app
 USER openslides
 RUN curl -o- -L https://yarnpkg.com/install.sh | bash
 RUN $HOME/.yarn/bin/yarn --non-interactive
+RUN node_modules/.bin/gulp --production
```

Comment:
Are you sure, that you should run gulp before getting the python requirements?

Reply:
Yes, it is independet in which order your run the setup (yarn or  pip install).